.travis.yml

```diff
@@ -21,7 +21,7 @@ script:
     echo "Updated: ${envs[@]}"
 
     for dir in "${envs[@]}"; do
-      man=$dir/manifest.json; etl=$dir/etlMapping.yaml
+      man=$dir/manifest.json; etl=$dir/etlMapping.yaml; ptlcfg=$dir/portal/gitops.json;
```

Comment:
please also move the log like in QA https://github.com/uc-cdis/gitops-qa/pull/1282/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R24